sys/net/crosslayer/ng_pktbuf/_pktbuf_static.c

```diff
@@ -165,7 +168,7 @@ void *_pktbuf_internal_alloc(size_t size)
 
     while ((node->next != NULL)
            /* and if space between current and next allocation is not big enough */
-           && ((_start_idx(node->next) - _end_idx(node)) < _total_sz(size))) {
+           && ((_start_idx(node->next) - _end_idx(node)) < _al_sz(_total_sz(size)))) {
```

Comment:
Thanks for the detailed feedback! Last one:

> A new node/chunk (a node just describes a chunk that comes immediately after it so the terms are somewhat interchangable) is always created when this function is called

Which function exactly or in which line?